docs/reference/api.rst

```diff
@@ -239,6 +245,44 @@ API
         ``tickit.devices.source``
         -----------------------------
 
+    .. automodule:: tickit.devices.cryostream
+
+    ``tickit.devices.cryostream``
+    -----------------------------
+
+        .. automodule:: tickit.devices.cryostream.base
+            :members:
+            
+            ``tickit.devices.cryostream.base``
+            ----------------------------------
+        
+        .. automodule:: tickit.devices.cryostream.cryostream
+            :members:
+
+            ``tickit.devices.cryostream.cryostream``
+            ----------------------------------------
+        
```

Comment:
`states` & `status` should be included, even if we can't point to the `Enum`s from other places

Reply:
They should now be included.